persistence/src/test/resources/application.conf

```diff
@@ -3,7 +3,6 @@ akka {
   actor.warn-about-java-serializer-usage = off
 }
 
-akka.persistence.journal.plugin = "akka.persistence.journal.inmem"
-akka.persistence.snapshot-store.plugin = "akka.persistence.snapshot-store.local"
-akka.persistence.snapshot-store.local.dir = "target/snapshots"
+akka.persistence.journal.plugin = "inmemory-journal"
```

Comment:
This looks very similar to the test config, but the test config has `akka.persistence.query.journal.id = "inmemory-read-journal"` and this one doesn't. Why?